Helper/Data.php

```diff
@@ -36,9 +36,9 @@ class Data extends AbstractHelper
     const DHL_SPECIFIC_COUNTRY_PATH = "payment/reach_payment/reach_dhl/specificcountry";
     const DHL_OPT_SPECIFIC          = "payment/reach_payment/reach_dhl/optional_allowspecific";
     const DHL_OPT_SPECIFIC_COUNTRY  = "payment/reach_payment/reach_dhl/optional_specificcountry";
-    const DHL_KEY_PATH              = "payment/reach_payment/reach_dhl/dhl_key";
+    const DHL_CLIENT_ID_PATH        = "payment/reach_payment/reach_dhl/dhl_client_id";
     const DHL_SHIPPING_PATH         = "payment/reach_payment/reach_dhl/applicable_shipping";
-    const DHL_SECRET_PATH           = "payment/reach_payment/reach_dhl/dhl_api_secret";
+    const DHL_CLIENT_SECRET           = "payment/reach_payment/reach_dhl/dhl_client_secret";
```

Comment:
should this be DHL_CLIENT_SECRET_PATH ? just to stay consistent.  Although if that's the case, we should probably fix DHL_OPT_SPECIFIC and DHL_OPT_SPECIFIC_COUNTRY, what do you think?